### ISSUE TITLE
Sound.loop is broken in excalibur 0.27

### ISSUE THREAD

**Issue by djcsdy** [excaliburjs/Excalibur#2523]
### Steps to Reproduce

```js
// Assuming sound is a loaded instance of Sound
sound.loop = true;
sound.play();
```

### Expected Result

Sound plays and loops continuously.

### Actual Result

Sound plays once then stops.

### Environment

- browsers and versions: Firefox 107, Chrome 107
- operating system: Windows 10
- Excalibur versions: 0.27.0
- Works as expected on at least 0.26.0, 0.25.3, 0.24.4

### Current Workaround

None known.

**Comment by eonarheim** [comment]
Hi @djcsdy thanks for the issue! I think this is fixed in the lastest alpha build of Excalibur. 

Let me know if that doesn't work for you https://github.com/excaliburjs/Excalibur/commit/5c968673cdeb5bd43457eac7bc31bf8c7334872b

We'll be trying to get a new version of Excalibur cut before the end of the year hopefully🤞

**Comment by eonarheim** [comment]
Apologies didn't post the version, I think the latest alpha version should work https://www.npmjs.com/package/excalibur/v/0.28.0-alpha.589

**Comment by djcsdy** [comment]
It does indeed seem to be fixed in v0.28.0-alpha.589, thanks!

**Comment by KokoDoko** [comment]
The latest stable release still has the audio loop bug, I now use a regular `new Audio()` object to get looping audio.

**Comment by eonarheim** [comment]
@KokoDoko Thanks that's a great workaround until we ship the next version

**Comment by djcsdy** [comment]
@KokoDoko Unfortunately that workaround will introduce gaps between loops which might be acceptable for some use cases but isn't acceptable for mine. I'm still using 0.26.0 for now.

(back in the day when I made Flash games I went to the trouble of modifying the Flex compiler to get rid of gaps between looped MP3s, I *really* care about this :-))

**Comment by eonarheim** [comment]
@djcsdy We'll get a new version out soon with the fix I promise ❤️

**Comment by eonarheim** [comment]
Fixed in v0.28.0! Just released!
